Makefile

```diff
@@ -55,10 +58,10 @@ DYNAMICCLIB=lib/$(CLIBNAME).so
 
 HEADERS = src/cufinufft.h src/deconvolve.h src/memtrasfer.h src/profile.h \
 	src/spreadinterp.h
-FINUFFTOBJS=finufft/utils.o finufft/dirft2d.o finufft/common.o \
-	finufft/spreadinterp.o finufft/contrib/legendre_rule_fast.o
+CONTRIBOBJS=contrib/utils.o contrib/dirft2d.o contrib/common.o \
+	contrib/spreadinterp.o contrib/legendre_rule_fast.o
 
-CUFINUFFTOBJS=src/2d/spreadinterp2d.o src/2d/cufinufft2d.o \
+CUCONTRIBOBJS=src/2d/spreadinterp2d.o src/2d/cufinufft2d.o \
```

Comment:
These should still be `CUFINUFFTOBJS`, no?

Reply:
Yes, this was a mistake. Thanks for pointing this out.